lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -386,6 +386,25 @@ walletStateOptions = taggedSumTypeOptions $ TaggedObjectOptions
     , _contentsFieldName = "progress"
     }
 
+{-------------------------------------------------------------------------------
+                             FromText/ToText instances
+-------------------------------------------------------------------------------}
+
+instance FromText AddressAmount where
+    fromText text = do
+        let err = Left . TextDecodingError $ "Parse error. Expecting format\
+            \'<amount>@<address>' but got '" <> show text <> "'"
```

Comment:
There's a missing space after `format`, and `show text` will already produce a string surrounded by double quotes, so, the extra single quotes around are superfluous. For consistency, I'd also change the quotes around `<amount>@<address>` to be double-quotes.

Reply:
good catch!